c_src/build_deps.sh

```diff
@@ -56,17 +56,24 @@ case "$1" in
 
     get-deps)
         if [ ! -d leveldb ]; then
-            git clone git://github.com/basho/leveldb
+            git clone https://github.com/basho/leveldb
             (cd leveldb && git checkout $LEVELDB_VSN)
-            if [ "$BASHO_EE" = "1" ]; then
-                (cd leveldb && git submodule update --init)
-            fi
+            (cd leveldb && git submodule update --init)
         fi
         ;;
 
     *)
-        export MACOSX_DEPLOYMENT_TARGET=10.8
+        TARGET_OS=`uname -s`
 
+        export CFLAGS="$CFLAGS -I $BASEDIR/system/include"
+        export CXXFLAGS="$CXXFLAGS -I $BASEDIR/system/include"
+        # On GCC, we pick libc's memcmp over GCC's memcmp via -fno-builtin-memcmp
+        if [ "$TARGET_OS" = "Darwin" ]; then
+            export MACOSX_DEPLOYMENT_TARGET=10.8
```

Comment:
It doesn’t matter much, but, anyone knows what that is about?  10.8 is Mountain Lion (so 2012).